### PR TITLE
Add Google Photos Backup.app v1.1.0.261

### PR DESCRIPTION
**Pull request by mjgardner**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There’s trailing whitespace that needs to be removed.

File: Casks/google-photos-backup.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on :macos => '>= :lion'
+
+  zap :delete => [
+                  '~/Library/Application Support/Google/Google Photos Backup',
+                  '~/Library/Caches/com.google.gpautobackup',
+                  '~/Library/Preferences/com.google.gpautobackup.plist',
+                 ],
+      :rmdir  => '~/Library/Application Support/Google'
+end
+
```
Comment (vitorgalvao) [inline Casks/google-photos-backup.rb]: This line must go.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/google-photos-backup.rb (hunk 1 above)
Comment (mjgardner) [inline Casks/google-photos-backup.rb]: Fixed in #11449

**Pull request closed without merging**